cmd/bugsnag/release.go

```diff
@@ -0,0 +1,156 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+
+	"github.com/kinbiko/bugsnag/builds"
+)
+
+type releaseFlags struct {
+	apiKey            *string
```

Comment:
Add a comment saying that these will be empty values until the parsing is complete, and thus can't be set as values directly, but have to remain pointers.